src/frontend/src/pages/myfeeds.js

```diff
@@ -90,24 +91,31 @@ export default function MyFeeds() {
   }, [telescopeUrl, userInfo]);
 
   useEffect(() => {
-    if (userInfo.id || submitStatus.isError === false) {
-      (async function hashUserFeeds() {
-        try {
-          const userFeeds = await getUserFeeds();
-
-          const userFeedHash = userFeeds.reduce((hash, feed) => {
-            hash[feed.id] = { author: feed.author, url: feed.url };
-            return hash;
-          }, {});
-
-          updateFeedHash(userFeedHash);
-        } catch (error) {
-          console.log('Error hashing user feeds', error);
-          throw error;
-        }
-      })();
+    console.log('feedHash updated, setting numFeeds to', Object.keys(feedHash).length);
```

Comment:
Let's dump this

Reply:
Done in https://github.com/Seneca-CDOT/telescope/pull/994/commits/4aa793648200abf1b74f2034793548487cf486db